include/Reader/readerir.h

```diff
@@ -558,6 +560,12 @@ class GenIR : public ReaderBase {
 
   EHRegion *rgnAllocateRegion() override;
   EHRegionList *rgnAllocateRegionList() override;
+  void setDebugLocation(uint32_t CurrOffset, uint32_t IsCall) override;
```

Comment:
I can do IsStackEmpty, but we only set debug locations if StackEmpty is true or if it's a call instruction. So I thought the one bool would cover it.